### ISSUE TITLE
Feature request: add default value to var_lab

### ISSUE THREAD

**Issue by DanChaltiel** [gdemin/expss#60]
Hi Gregory,

I used to use `Hmisc` to deal with labels, but I now think that `expss` is far more complete.

However, there is one feature that I miss: default values.

```r
x=1:10
y=10:20

library(expss)
var_lab(x) = "I am X"
var_lab(x)
var_lab(y)

library(Hmisc)
label(x)
label(y)
label(y, default="I am Y")
```

Would you consider a pull request adding this feature?

**Comment by gdemin** [comment]
Hi Dan,
Yes, I will accept your pull request

**Comment by gdemin** [comment]
Released in version 0.10.5
